### ISSUE TITLE
debian packaging support

### ISSUE THREAD

**Issue by sigmonsays** [stedolan/jq#1039]
would a patch for building debian package be accepted?

**Comment by dtolnay** [comment]
Yes! @else may be interested - he has been most heavily involved in Debian packaging.

**Comment by else** [comment]
Is this about adding the necessary files to build a Debian package? I'd be interested to see it, but please note that jq is already available as Debian package. See http://tracker.debian.org/jq for more information.

If you have any suggestions for improving the source package, please file a bug against http://bugs.debian.org/source:jq

Thanks!

**Comment by sigmonsays** [comment]
Yes, I would just like to git clone and be able to build a package. So putting the necessary debian files in place from the source package makes sense to me.

**Comment by sigmonsays** [comment]
created pull #1058 (https://github.com/stedolan/jq/pull/1058)

I grabbed the source package that @else mentioned
